src/psyclone/tests/psyir/transformations/kernel_transformation_test.py

```diff
@@ -32,8 +32,9 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Author: A. R. Porter, STFC Daresbury Lab
-# Modified by: R. W. Ford, STFC Daresbury Lab,
+# Modified by: R. W. Ford, STFC Daresbury Lab
 #              I. Kavcic, Met Office.
```

Comment:
It's a tiny thing but it bothers me so please could you remove the full stop from this line.

Reply:
Done